manual_jb/content/analyses/water_facility_restoration.md

```diff
@@ -0,0 +1,61 @@
+# Water facility restoration
+
+**Description**
+
+This analysis computes the repair time, and the percentage of functionality change with time for water facility 
+restoration based on a particular hazard. Currently supported hazard is: **earthquake**.
```

Comment:
I think the supported hazards should the ones supported by Water facility damage. I don't see anything specific to EQ in this analysis except we are using an eq specific mapping from the database. Tagging @navarroc for his input

Reply:
oh i put down earthquake cuz that's the curves I have right now. Not sure what it should be

Reply:
Are the restoration curves specific to earthquake? The curves look to be in the form of X amount of damage it takes Y time to repair so it seems generic and could take any water facility damage we could support.

Reply:
I put this in pipeline analysis. if that's acceptable we can use similar format for other restoration analyses
```
This analysis computes water pipeline restoration time after a particular hazard. The analysis itself is hazard agnostic, as it works
 with the output of [Pipeline damage with repair rate analysis](pipeline_dmg_w_repair_rate.md) and restoration curves.
```